Reason: We only want to build man pages, not development docs
Upstream: An issue has been filed - https://gitlab.freedesktop.org/NetworkManager/NetworkManager/-/issues/689
Index: b/meson.build
===================================================================
--- a/meson.build
+++ b/meson.build
@@ -943,10 +943,8 @@ if enable_qt
 endif
 
 if enable_docs
-  assert(enable_introspection, '-Ddocs=true requires -Dintrospection=true')
   assert(meson.version().version_compare('>= 0.46.0'), '-Ddocs requires meson >= 0.46')
   subdir('man')
-  subdir('docs')
 endif
 
 configure_file(
Index: b/clients/common/meson.build
===================================================================
--- a/clients/common/meson.build
+++ b/clients/common/meson.build
@@ -28,7 +28,7 @@ libnmc_base_dep = declare_dependency(
   link_with: libnmc_base,
 )
 
-if enable_docs
+if enable_docs and enable_introspection
   settings_docs_source = custom_target(
     'settings-docs.h',
     input: nm_settings_docs_xml_gir,
